lib/xwax/timecoder.c

```diff
@@ -96,7 +96,7 @@ static struct timecode_def timecodes[] = {
         .seed = 0x134503,
         .taps = 0x041040,
         .length = 1500000,
-        .safe = 605000,
+        .safe = 1480000,
```

Comment:
are these changes to the safe value from him or from me?  If he's made the values bigger we might be able to delete our patch.

Reply:
From digging through the history Mark has never changed these values. However it looks like I have messed up the patch. 
I will redo it.

Reply:
Now the patch is fixed. It was a wrong conflict resolving: 

https://github.com/mixxxdj/mixxx/blob/25d17b1ea31873289bea16e6b0c07a75d8d45dc6/lib/xwax/0003-Use-my-the-safe-values-I-found-work-better.patch

Can you remember why you have changed these values originaly?

Reply:
These values determine how much of the vinyl signal is in the very outside of the physical record.  On the records I tested, there's a good ten seconds while the needle is still in the very outer groove, which is where the dj drops the needle. The default values meant that dropping the needle in this part of the record would seek to a few seconds into the track.  I increased the values to make the experience better.